### DISCUSSION TITLE
only run httpd - "no service configured, nothing to do"

### DISCUSSION THREAD

**Discussion by JonBendtsen** [drakkan/sftpgo#1151] [Q&A]
Hi

I tried running only the httpd, disabling the SFTP and FTP, but I got this error message `no service configured, nothing to do` even after I also `enable_web_client": true,` but in reality I only want the admin interface so I can use the GUI and API to make changes in the content of the DB, which I can then share into other containers read only.

My particular json file does not have a ftpd segment, but it does have a sftpd segment which has
```
  "sftpd": {
    "bindings": [
      {
        "port": 0,
```

Can I run only the httpd service or will I have to enable a dummy unreachable sftpd/ftpd service?

**Comment by drakkan** [comment]
https://github.com/drakkan/sftpgo/commit/496c8bc785fc5c4eb77346cdc4f4e7bc70701e27
